### DISCUSSION TITLE
Doc update - Ventura spotlight  no longer offer markdown preview,  refer to external app

### DISCUSSION THREAD

**Discussion by danieltomasz** [chrisgrieser/shimmering-obsidian#126] [General]
in Ventura spotlight no longer offer preview for md files,

installing this app solves it it, maybe useful to put it in the documentation  (there also exists sister app for code files, including tex from the same developer)
https://github.com/sbarex/QLMarkdown

I recommend  to do some tweaks after installing though
- enabling the advanced settings
- switching to source view of the file
- use fixed quick look window size, the default on was way too big for me
![image](https://user-images.githubusercontent.com/7980381/232233849-f5f9c5c4-3896-496e-b4e7-38f147c5711d.png)

**Comment by chrisgrieser** [comment]
already part of the docs: https://github.com/chrisgrieser/shimmering-obsidian/blob/main/docs/Alfred-based%20Quick%20Switcher.md#previewing-notes-via-quicklook

**Comment by yarub123** [comment]
Have any of you figured out how to show images? I've been using QLmarkdown for a while now, and it seems it still has an issue of not showing up the images. Idk if this has to do with the path or the fact that they're embedded wikilinks with this notation:

<img width="443" alt="image" src="https://user-images.githubusercontent.com/84487964/234583318-5f9dd3f0-d168-4f4b-a081-0818a02ee7f1.png">

I looked into this issue way before this (Shimmering) plugin came out and it seems that there is no workaround as of now (?)

Oh well, not a huge deal. Was just curious. 
Cheers.

Edit:
wanted to clarify, Screenshot images (copy and paste) don't show up, but drag and drop, they do show up since they are external links.

**Reply by chrisgrieser** [reply]
showing images in the preview won't work at all, since the `![[ ]]` is specific to Obsidian and not markdown standard. 

What might work – I haven't tried – is if you use the standard markdown syntax for images like `![]()`
